portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path

```diff
@@ -230,6 +230,26 @@
 	<td>//div[label[contains(.,'Source Options')]]/select</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRESELECT_VALUE</td>
+	<td>//button[@class="dropdown-item"]//strong</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SELECT_FILTER_DATE</td>
+	<td>//div[contains(@class,"dropdown-menu date-picker-dropdown-menu show")]//select[contains(@class,'form-control form-control-sm') and contains(@name,'${key_type}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CHOOSE_DATE</td>
+	<td>//div[contains(@class,"form-group-item") and contains(.,'${key_field}')]//button[@aria-label="Choose date"]</td>
+	<td></td>
+</tr>
```

Comment:
This path seems to be pretty general, can you use the general dropdown paths already created in Poshi instead?

Also,"CHOOSE" sounds weird, put "SELECT" instead

Reply:
Done!